core/src/main/web/app/mainapp/components/notebook/codecell-directive.js

```diff
@@ -25,7 +45,8 @@
       bkSessionManager,
       bkCoreManager,
       bkPublicationHelper,
-      $timeout) {
+      $timeout,
+      $animate) {
```

Comment:
why do we need animate here? It seems that it's used only in ngHideEx directive.